### PR TITLE
[BASICALLY MODULAR] Runs grep checks on modular_skyrat and fixes all issues

### PR DESCRIPTION
**Pull request by Tastyfish**
## About The Pull Request

Modular except addition to ci_suite. Since we manually mirror that anyway, whatever.

Makes the check greps actually check modular_skyrat by some mild hackery of a single sed replacement. If check_grep ever changes, I can update the single line of code.

More substantially, I fixed all of the errors caught by check_grep. It's mostly whitespace and formatting errors. Amusingly, I found a couple of mild problems like comments misspelling files and a single case of code actually being wrong due to bad whitespace on the dance machine.

Have fun reviewing this, suckers.

## How This Contributes To The Skyrat Roleplay Experience

CI

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
</details>

## Changelog

Not player-facing

### CODE REVIEW

File: modular_skyrat/modules/novaya_ert/code/advanced_choice_beacon.dm [hunk 1]
```diff
@@ -153,7 +153,7 @@
 /obj/structure/closet/crate/secure/weapon/nri/engineer/defense
 	name = "defensive engineering supplies"
 	loadout_desc = "An assortment of engineering supplies finely tuned for quick fortification.\
-	 Features barricades, building materials, extra large fuel tank and 5.6mm defensive autoturrets."
+		Features barricades, building materials, extra large fuel tank and 5.6mm defensive autoturrets."
```
Comment (Zonespace27) [inline modular_skyrat/modules/novaya_ert/code/advanced_choice_beacon.dm:156]: Does this spacing work in-game without issue?

File: modular_skyrat/modules/novaya_ert/code/advanced_choice_beacon.dm [hunk 2]
```diff
@@ -178,7 +178,7 @@
 /obj/structure/closet/crate/secure/weapon/nri/engineer/offense
 	name = "offensive engineering supplies"
 	loadout_desc = "An assortment of engineering supplies finely tuned for rapid approach defortification and area suppression.\
-	 Features way less barricades and building materials than its more defensive analogue, but includes NRI-issued viscerator grenades and a combat RCD."
+		Features way less barricades and building materials than its more defensive analogue, but includes NRI-issued viscerator grenades and a combat RCD."
```
Comment (Zonespace27) [inline modular_skyrat/modules/novaya_ert/code/advanced_choice_beacon.dm:181]: Same goes for this and other multiline strings

File: modular_skyrat/modules/verbs/code/say.dm [hunk 3]
```diff
@@ -1,11 +1,5 @@
 /mob/proc/get_top_level_mob()
-	if(istype(src.loc,/mob)&&src.loc!=src)
-		var/mob/M=src.loc
+	if(istype(loc, /mob) && loc != src)
```
Comment (Zonespace27) [inline modular_skyrat/modules/verbs/code/say.dm]:
```suggestion
	if(ismob(loc) && (loc != src))
```
Might as well since we're here

**Review by Zonespace27 — COMMENTED** [review]
Not seeing any regressions

File: modular_skyrat/modules/novaya_ert/code/advanced_choice_beacon.dm (hunk 1 above)
Comment (Tastyfish) [reply modular_skyrat/modules/novaya_ert/code/advanced_choice_beacon.dm:156]: The space between the sentences is lost, I likely can remedy this by putting it at the end of the previous line, one moment.

Comment (Tastyfish) [reply modular_skyrat/modules/novaya_ert/code/advanced_choice_beacon.dm:156]: Fixed.

File: modular_skyrat/modules/novaya_ert/code/advanced_choice_beacon.dm (hunk 2 above)
Comment (Tastyfish) [reply modular_skyrat/modules/novaya_ert/code/advanced_choice_beacon.dm:181]: Fixed.

**Review by Zonespace27 — APPROVED** [review]

File: .github/workflows/ci_suite.yml [hunk 4]
```diff
@@ -41,6 +41,7 @@ jobs:
           bash tools/ci/check_filedirs.sh tgstation.dme
           bash tools/ci/check_changelogs.sh
           bash tools/ci/check_grep.sh
+          bash skyrat/tools/skyrat_check_grep.sh
```
Comment (GoldenAlpharex) [inline .github/workflows/ci_suite.yml]: Is there a way to add comments in these files? If so, I'd appreciate a `// SKYRAT EDIT - Running the Skyrat Check grep` comment (not necessarily phrased like this, but you know what I mean), to save us some time later when there's inevitably conflicts in this file.

File: modular_skyrat/modules/altborgs/code/robot_model.dm [hunk 5]
```diff
@@ -324,14 +324,14 @@
 	)
 
 /obj/item/robot_model/syndicatejack/rebuild_modules()
-    ..()
-    var/mob/living/silicon/robot/syndicatejack = loc
-    syndicatejack.scrambledcodes = TRUE // We're rouge now
+	..()
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/altborgs/code/robot_model.dm]:
```suggestion
	. = ..()
```

File: modular_skyrat/modules/altborgs/code/robot_model.dm [hunk 6]
```diff
@@ -324,14 +324,14 @@
...
+	var/mob/living/silicon/robot/syndicatejack = loc
+	syndicatejack.scrambledcodes = TRUE // We're rouge now
 
 /obj/item/robot_model/syndicatejack/remove_module(obj/item/I, delete_after)
-    ..()
-    var/mob/living/silicon/robot/syndicatejack = loc
-    syndicatejack.scrambledcodes = FALSE // Friends with the AI again
+	..()
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/altborgs/code/robot_model.dm]:
```suggestion
	. = ..()
```

File: modular_skyrat/modules/customization/modules/language/xerxian.dm [hunk 7]
```diff
@@ -6,7 +6,7 @@
 	syllables = list(
 		"a", "ah", "ba", "lah", "tha", "shah", "sa", "cha", "zah", "in", "al", "an",
 		"i", "ih", "bi", "lih", "thih", "shih", "sih", "chi", "zih", "ji", "nn", "jo", "ja",
-        "o", "oh", "boh", "loh", "tho", "sho", "so", "cho", "zo", "ya", "yi", "yo"
+		"o", "oh", "boh", "loh", "tho", "sho", "so", "cho", "zo", "ya", "yi", "yo"
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/language/xerxian.dm]:
```suggestion
		"o", "oh", "boh", "loh", "tho", "sho", "so", "cho", "zo", "ya", "yi", "yo",
```

File: modular_skyrat/modules/modular_items/code/modular_glasses.dm [hunk 8]
```diff
@@ -4,222 +4,222 @@
...
+		flash_protect = (mode == MODE_OFF_FLASH_PROTECTION) ? FLASH_PROTECTION_FLASH : FLASH_PROTECTION_NONE /// when off is supposed to have flash protection
+		tint = (mode == MODE_OFF_FLASH_PROTECTION) ? 0 : 1 /// when off is suppost to tint the glasses
+		vision_flags = 0 /// Sets vision_flags to 0 to disable meson view mainly
+		lighting_alpha = user.default_lighting_alpha() // Resets lighting_alpha to user's default one
+		hud_type = null /// no hud when off, obviously!
+		hud_trait = null /// no hud when off, obviously!
+		clothing_traits = null /// also disables the options for Science functionality
+		REMOVE_TRAIT(user, hud_trait, GLASSES_TRAIT)
+	playsound(src, modeswitch_sound, 50, TRUE) // play sound set in vars!
+	// Blah blah, fix vision and update iconstoggle_mode
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/modular_items/code/modular_glasses.dm]: Was this comment change intended?

File: modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/shackles.dm [hunk 9]
```diff
@@ -84,15 +84,15 @@
...
-        var/obj/item/clothing/suit/straight_jacket/shackles/reinforced/shackles = new /obj/item/clothing/suit/straight_jacket/shackles/reinforced
-        remove_item_from_storage(user)
-        user.put_in_hands(shackles)
-        to_chat(user, span_notice("You reinforced the locks on [src] with [used_item]."))
-        qdel(used_item)
-        qdel(src)
-        return
-    . = ..()
+	if(istype(used_item, /obj/item/restraints/handcuffs))
+		var/obj/item/clothing/suit/straight_jacket/shackles/reinforced/shackles = new /obj/item/clothing/suit/straight_jacket/shackles/reinforced
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/shackles.dm]: I don't remember, can't we just remove what's after that `new` to have DM just infer the type to spawn from the type the variable is casted as?

File: modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/shackles.dm [hunk 10]
```diff
@@ -84,15 +84,15 @@
...
+		remove_item_from_storage(user)
+		user.put_in_hands(shackles)
+		to_chat(user, span_notice("You reinforced the locks on [src] with [used_item]."))
+		qdel(used_item)
+		qdel(src)
+		return
+	. = ..()
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/shackles.dm]:
```suggestion

	return ..()
```

File: modular_skyrat/modules/modular_items/lewd_items/code/lewd_items/feather.dm [hunk 11]
```diff
@@ -71,9 +71,12 @@
 	target.add_mood_event("tickled", /datum/mood_event/tickled)
 	target.adjust_arousal(3)
 	user.visible_message(span_purple("[user] [message]!"))
-	playsound(loc, pick('sound/items/handling/cloth_drop.ogg', // I duplicate this part of code because im useless shitcoder that can't make it work properly without tons of repeating code blocks
-            			'sound/items/handling/cloth_pickup.ogg', // If you can make it better - go ahead, modify it, please.
-        	       	    'sound/items/handling/cloth_pickup.ogg'), 70, 1, -1, ignore_walls = FALSE)
+	playsound(loc, \
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/modular_items/lewd_items/code/lewd_items/feather.dm:74]: What's the `\` for?

File: modular_skyrat/modules/singularity_engine/code/particle_accelerator/particle_accelerator.dm [hunk 12]
```diff
@@ -1,14 +1,15 @@
...
+	3 Particle Emitters
+	1 Power Box
+	1 Fuel Chamber
+	1 End Cap
+	1 Control computer
+	Setup map
+		|EC|
+	CC|FC|
+		|PB|
+	PE|PE|PE
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/singularity_engine/code/particle_accelerator/particle_accelerator.dm]: Somehow something tells me like that's not quite the display we want this to have, unless it displays differently in VSC.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
This is some good shit right there.

File: .github/workflows/ci_suite.yml (hunk 4 above)
Comment (Tastyfish) [reply .github/workflows/ci_suite.yml]: Not home right now, but yeah I can add a comment later

File: modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/shackles.dm (hunk 9 above)
Comment (Tastyfish) [reply modular_skyrat/modules/modular_items/lewd_items/code/lewd_clothing/shackles.dm]: Yes

File: modular_skyrat/modules/modular_items/code/modular_glasses.dm (hunk 8 above)
Comment (Tastyfish) [reply modular_skyrat/modules/modular_items/code/modular_glasses.dm]: nop

**Comment by Tastyfish** [comment]
@GoldenAlpharex All of your suggestions were committed/fixed.

**Review by GoldenAlpharex — APPROVED** [review]
Nice.

**Pull request merged**